contracts-periphery/Makefile

```diff
@@ -6,8 +7,12 @@ else
   INSTALL_CMD := forge install
 endif
 
+forkBlock = 14800081
+
 build :; forge build
 clean :; forge clean
 install :; $(INSTALL_CMD)
 snapshot :; forge snapshot --match-path test/UmbraBatchSend.gas.t.sol
-test :; forge test
\ No newline at end of file
+test :; forge test --no-match-path test/UniswapWithdrawHook.fork-1.t.sol && make test-hook
+test-hook :; forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path test/UniswapWithdrawHook.fork-1.t.sol --match-test "testFuzz_HookTest_10Addrs"
+test-hook-50 :; forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path test/UniswapWithdrawHook.fork-1.t.sol --match-test "testFuzz_HookTest_50Addrs"
```

Comment:
I think we should clean up the make file commands a bit here:
- There's no command to just run the `UniswapWithdrawHook` tests
- Having `test-hook` and `test-hook-50` is a bit clunky—the better way to do this is with standard fuzz tests, and change the number of fuzz runs by choosing a different profile or by using the `FOUNDRY_FUZZ_RUNS` env var (more info on both in the foundry book)

Reply:
Implemented env variable in 70e55ca3ade277cc79101d7dc61ddabb7b022e23

Reply:
So CI is failing because we have to set a github env var, but what do you think about just using `setEnv` in `Setup` instead since it's not really a secret we need to hide.

Reply:
But then might as well just create a `uint256` variable inside the function or `setUp` that specifies how many addrs needs to be fuzzed over... so it seems like there are many options, which would you recommend?

Reply:
In this particular case, I don't think we need to stress test the number of addresses we cover in the fuzzer since that's a straightforward thing to test, so I don't want to overcomplicate it. Your idea of a `uint256` state variable in the test contract is simple and works. We can just hardcode something small like 10 which should be sufficient.

And just make sure to add some comments explaining why we're using a for loop with manually generated addresses instead of a fuzz param

Reply:
Sounds good, addressed it here https://github.com/ScopeLift/umbra-protocol/pull/341/commits/c0e8d99f787cb8502889889b8fb7dfcfdd216833

Reply:
> Also bumping a comment/gas framework here in case you missed it https://github.com/ScopeLift/umbra-protocol/pull/341#issuecomment-1155652628

Ah yes I saw. Since it's not a thread there's nothing to resolve, but just gave it a thumbs up to make acknowledgement explicit—sorry!